go/api/response_handlers.go

```diff
@@ -31,6 +32,12 @@ func handleStringOrNullResponse(response *C.struct_CommandResponse) string {
 	return handleStringResponse(response)
 }
 
+// handleBooleanResponse converts a C struct_CommandResponse's bool_value to a Go bool.
+func handleBooleanResponse(response *C.struct_CommandResponse) bool {
```

Comment:
Can we have the methods consistent please?

E.g.:

Either 

- getXXXResponse returns XXX OR
- getXXXResponse returns XXX, err
-

Reply:
@ravisi21 It's done.